palantirnet/federated-search-react
# Make the React app render in IE-11

[UMHS-22-Unified search-- search page does not render in IE](https://palantir.atlassian.net/browse/UMHS-22)


Testing Instructions

- Create a src/.env.local.js configuration file by copying the src/.env.local.js.example.
- run 'yarn install' and `yarn start`
- visit browserStack (credentials in 1pass) and check that the app renders in IE-11
- Also check that the console does not throw any error.
